docs/data/versions.json

```diff
@@ -0,0 +1,13 @@
+{
+	"params": {
+		"currentVersion": ["2.0"],
+		"ltsVersion": ["1.0"],
+		"previousVersions": ["1.0"]
+	},
+	"apiDocContentPath": {
+		"Select version": "/docs/apis/",
+		"Fluid Framework 1.0": "/docs/apis/1.0/",
+		"Fluid Framework 2.0 Alpha": "/docs/apis/2.0/"
+	},
+	"apiDocIndexPath": ["/docs/apis/1.0/_index.md", "/docs/apis/2.0/_index.md"]
```

Comment:
I don't think we need to have this extra metadata (apiDocsContentPath and apiDocsIndexPath) for the paths here. We can always derive `docs/apis/<version>` from the version number we're looking at.

Reply:
If it's useful (and it might be), we could add an `apiDocsDirectory` (or something) value that just points to `docs/apis`. Then we could use that plus the version numbers to derive the paths you need, without having to hard-code `docs/apis` all over the place in the Hugo code.

Reply:
Updated.